dist/index.js

```diff
@@ -57,13 +57,8 @@ function koaDevware(compiler, options) {
 
   function middleware(context, next) {
     return new _promise2.default(function (resolve, reject) {
-      // https://github.com/webpack/docs/wiki/plugins#donestats-stats
-      compiler.plugin('done', function (stats) {
-        resolve(stats);
-      });
-
-      compiler.plugin('failed', function (error) {
```

Comment:
Let's say that I setup the webpack configuration to dynamically serve /js/bundle.js which is referenced from index.html which is statically served from another middleware. I found out that if I do a curl for localhost:<somePort>/js/bundle2.js which webpack isn't setup to dynamically serve nothing happens, no response is returned, no 404, no nothing....